WMFData/Sources/WMFData/Data Controllers/Year In Review/WMFYearInReviewSlideProtocol.swift

```diff
@@ -0,0 +1,352 @@
+import Foundation
+import CoreData
+
+public protocol YearInReviewSlideProtocol {
```

Comment:
I think the protocol should be in it's own file.
This file is named `YearInReviewSlideProtocol`, but it also contains other models, creates each of the slides, etc.
It'll be hard to find what we need in the future, so maybe splitting the files to make it clearer will help